runtimevar/runtimeconfigurator/runtimeconfigurator.go

```diff
@@ -168,7 +168,17 @@ func (o *URLOpener) OpenVariableURL(ctx context.Context, u *url.URL) (*runtimeva
 	}
 	var rn ResourceName
 	rn.ProjectID = u.Host
-	rn.Config, rn.Variable = path.Split(u.Path)
+	path := u.Path
+	if len(path) > 0 && path[0] == '/' {
```

Comment:
Instead, you can do
```go
if len(path) > 0 {
  path = strings.TrimLeft(path, "/")
}
```
if you want to strip all leading "/"s.

Reply:
Done. The `len(path)>0` isn't needed then either.